src/conn/conn_tiered.c

```diff
@@ -64,11 +64,6 @@ __tier_storage_remove_local(WT_SESSION_IMPL *session)
               session, WT_VERB_TIERED, "REMOVE_LOCAL: %s in USE, queue again", object);
             WT_STAT_CONN_INCR(session, local_objects_inuse);
             /*
-             * FIXME-WT-7470: If the object we want to remove is in use this is the place to call
```

Comment:
This is no longer relevant as the `WT_FH` array [has been removed](https://jira.mongodb.org/browse/WT-7470?focusedCommentId=4356010&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4356010).